.github/workflows/backend-build-test.yml

```diff
@@ -0,0 +1,26 @@
+name: Backend build and test
+
+on:
+  push:
+    paths:
+      - "backend/**"
+
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '8.0.x'
+
+      - name: Restore dependencies
+        run: dotnet restore backend/FormAPI.sln
+
+      - name: Build
+        run: dotnet build backend/FormAPI.sln --no-restore
+        
+      - name: Test
+        run: dotnet test backend/FormAPI.sln --no-build
```

Comment:
Same comment as on the other PR - might consider splitting these steps into two jobs for easier monitoring and analyzing if failure 🫣